### DISCUSSION TITLE
Deprecation

### DISCUSSION THREAD

**Discussion by oceandlr** [ovis-hpc/ldms#596] [General]
Things to look at deprecating (no urgency)

**Comment by oceandlr** [comment]
Cray_system_sampler - @oceandlr should look into this

**Comment by oceandlr** [comment]
@baallan Can V3 documentation be removed from the wiki menu?

**Comment by baallan** [comment]
Anything is possible, but v3 is still in use at snl/lanl/probably awe.

**Comment by oceandlr** [comment]
@tom95858 @baallan @valleydlr 

Once l2_streams_csv_store -> streams_csv_store gets pulled in, does anyone object to:
* deprecate sampler/hello_stream/hello_cat_publisher
* deprecate sampler/hello_stream/hello_publisher
Because these both have been obviated by ldmsd_stream_publish
* add a man page for ldmsd_stream_publish

and also:
* deprecate store/hello_stream/hello_stream_store
Because it has effectively been replaced by streams_csv_store (Differences in features are not used).
* rename store/hello_stream to store/streams

**Reply by baallan** [reply]
I think we're early enough in the life of these things that everything you just proposed deprecating should simply be removed. Lingering (un)maintained examples are just confusing and possibly a maintenance burden.
The repo is public-- if anyone really wants them, they can browse the old versions.

If any of those old drivers do something that is not covered by either stream_publish or stream_csv, those somethings should be incorporated in ldmsd/test/.

Adding (correct) man pages is always good.

**Reply by baallan** [reply]
I would also like to vote/request that we use 'stream' everywhere in file names instead of an arbitrary mixture of stream and streams.

**Reply by oceandlr** [reply]
This is all part of pull request #777
